.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: elixir
```

Comment:
I need to improve the test suite a little bit before running it on CI, could you remove this from the PR? But thanks for a reminder, it should definitely be there.

Reply:
there is no reason not to run the tests, even if they are not complete as long as they do not fail lets run it on Travis.